tests/basic_functionality.rs

```diff
@@ -35,6 +35,7 @@ fn generate_temporary_contact_numbers_and_report_them() {
 
     // Check that the recomputed TCNs match the originals.
     // The slice is offset by 1 because tcn_0 is not included.
+    println!("&tcns[20 - 1..90] len: {:?}", &tcns[20 - 1..90].len());
```

Comment:
I'd remove all these prints. I doubt that they'll be useful to other people running the tests.